config/main.py

```diff
@@ -64,39 +63,42 @@ def interface_name_to_alias(interface_name):
         for port_name in natsorted(port_dict.keys()):
             if interface_name == port_name:
                 return port_dict[port_name]['alias']
-        print "Invalid interface {}".format(interface_name)
-        sys.exit(0)
-
+        return None
 
 
 def set_interface_mode(mode):
-    """Modify IFMODE env variable in user .bashrc
+    """Modify SONIC_CLI_IFACE_MODE env variable in user .bashrc
     """
-    sudo_user = os.getenv('SUDO_USER')
-    user = os.getenv('USER')
-    set_mode = "IFMODE={}".format(mode)
-
-    if not sudo_user:
-        bashrc = "/{}/.bashrc".format(user)
+    user = os.getenv('SUDO_USER')
+    bashrc_ifacemode_line = "SONIC_CLI_IFACE_MODE={}".format(mode)
```

Comment:
I think this line is best moved down just above line 87, right before it is used. That will also keep all of the user name determination code together.